test/PHPMailer/DKIMTest.php

```diff
@@ -27,6 +27,13 @@ final class DKIMTest extends TestCase
 
     const PRIVATE_KEY_FILE = 'dkim_private.pem';
 
+    /**
+     * Whether or not to initialize the PHPMailer object to throw exceptions.
+     *
+     * @var bool|null
+     */
+    protected $exceptions = true;
```

Comment:
It seems very unlikely you'd ever want to change this, and there are unlikely to ever be multiple instances, so would it not be cleaner to make this static?

Reply:
Good point. What about making it a class constant ? And using it as `static::EXCEPTIONS` ?

Reply:
Sounds good to me.

Reply:
@Synchro Are you okay with me using "fixup" and force pushing the fixed commit ? That way the change is tidy in the commit history (but it is less easy to see what I changed since you last viewed the PR).

Reply:
Yes

Reply:
In that case, all done. I've also updated the PR description with the new commit information.